### ISSUE TITLE
Infiltrating squad bugged

### ISSUE THREAD

**Issue by imn01** [long-war-2/lwotc#314]
I'm having a problem with alpha 5.1. The issue is similar to #302 but exact opposite, so I'm opening a new issue. 

Say I have a premade squad(I made the squad in squad management) of 6 soldiers,  when sending them to a mission I remove 2 soldiers to get 100% infiltration. But after I start infiltrating I see (through view infiltrating squad) that whole squad is sent (all 6 including removed 2), this make all mission under infiltrated and impossible to do.

TL;DR: Started infiltration with the 4 and then I go check the squad that is infiltrating it then it shows all 6.

This can be easily reproduce, 
create a squad of 6+ soldiers >> select that squad to infiltrate >> remove 2+ soldiers >> start infiltration >> view infiltrating squad
